pyproject.toml

```diff
@@ -0,0 +1,118 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "timdex_dataset_api"
+version = "0.1.0"
```

Comment:
Read some docs on [Configuring setuptools using `pyproject.toml` files](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata) and sa that their base recommendation is to set the `version` dynamically s.t. you don't have to update the `pyproject.toml` when a new release is published. Seems like it can instead pull the version from a file (that would need to be updated) or it can derive it from the `git` version control system (i.e., GitHub release tag). Would be interested in exploring this as we get closer to an initial release!